payment/src/main/java/org/killbill/billing/payment/core/PaymentGatewayProcessor.java

```diff
@@ -77,12 +77,15 @@ public PaymentGatewayProcessor(final OSGIServiceRegistration<PaymentPluginApi> p
     }
 
     public GatewayNotification processNotification(final String notification, final String pluginName, final Iterable<PluginProperty> properties, final CallContext callContext) throws PaymentApiException {
+
         return dispatchWithExceptionHandling(null,
+                                             pluginName,
                                              new Callable<PluginDispatcherReturnType<GatewayNotification>>() {
                                                  @Override
                                                  public PluginDispatcherReturnType<GatewayNotification> call() throws PaymentApiException {
-                                                     final PaymentPluginApi plugin = getPaymentPluginApi(pluginName);
+
                                                      try {
+                                                         final PaymentPluginApi plugin = getPaymentPluginApi(pluginName);
```

Comment:
This is actually a mistake. I moved it out of the callable in the commit before and wanted to move it back in this commit. I have moved it out of the try block again now.